deps/patches/blastrampoline-cygwin_build_fix.patch

```diff
@@ -0,0 +1,39 @@
+From 4e78372de1cda6f5dfaaa25b6cea0abf73b57eb1 Mon Sep 17 00:00:00 2001
```

Comment:
I'm not sure if we should add a patch or update the version number (commit hash).

Reply:
So, will it be weeks or months before the new release of blastramploine?

Reply:
> I'm not sure if we should add a patch or update the version number (commit hash).

Updating the commit hash is beyond my skills.

Reply:
I thought for a moment that updating the version number directly might affect the normal build process. (Non-source compilation)
So it's better to use the patch.